src/wxflow/jinja.py

```diff
@@ -146,6 +147,7 @@ def get_set_env(self, loader: jinja2.BaseLoader, filters: Dict[str, callable] =
                 if not (isinstance(dt, SilentUndefined) or isinstance(delta, SilentUndefined))
                 else dt if isinstance(dt, SilentUndefined) else delta)
         env.filters["to_timedelta"] = lambda delta_str: to_timedelta(delta_str) if not isinstance(delta_str, SilentUndefined) else delta_str
+        env.filters["replace_tmpl"] = lambda string, tmpl_dict: replace_tmpl(string, tmpl_dict)
```

Comment:
```suggestion
        env.filters["replace_tmpl"] = lambda string, tmpl_dict: reduce(lambda s, k: s.replace(k, tmpl_dict[k]), tmpl_dict, string)
```
Import `reduce` from `functools`.